### ISSUE TITLE
Get more training data

### ISSUE THREAD

**Issue by freedomofkeima** [freedomofkeima/transfer-learning-anime#3]
The result are quite bad when the number of classes is increased to 100. However, more images still make quite a difference in detection quality. With 30 images, the overall accuracy is 60.3% while with around 40 images, the overall accuracy is 64.0%. For details, see [here](https://github.com/freedomofkeima/MoeFlow/blob/master/100_class_traning_note.md#results-additional).

One of the idea is to add more training data from animation video (screen capture).

**Comment by freedomofkeima** [comment]
Other ideas:
- Stability training: https://arxiv.org/pdf/1604.04326.pdf
- Fine-tune the last convolutional block: https://blog.keras.io/building-powerful-image-classification-models-using-very-little-data.html
- Features selection improvement (not limited to face detection only)
